### PR TITLE
MSQ files translated

### PR DESCRIPTION
**Pull request by allister-translator**
only 4 or 5 new lines, sasuga sega

### CODE REVIEW

File: Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (DabirSA) [inline Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv]: You seem to have accidentally blanked this file out.

Comment (allister-translator) [reply Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv]: how the fuck

Comment (DabirSA) [reply Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv]: And moved it to the wrong place, now that I look at it.

Comment (allister-translator) [reply Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv]: okay fixed

File: Translated/Story/08-09 Finding Bruder/msq_011360_000.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name_7006#0,"""Ran"""
+msq_011360_000_0010#0,"""Can you hear me? It looks like you've reached the Power Level designated for entry to the Resol Forest."""
+msq_011360_000_0020#0,"""Resol Forest is a gorgeous place, but there are many fearsome enemies lurking beneath its canope. Please be most careful."""
```
Comment (Frey8914) [inline Translated/Story/08-09 Finding Bruder/msq_011360_000.csv:3]: canopy

File: Translated/Story/08-09 Finding Bruder/msq_011418.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+msq_011418_0010#0,"""Before we go any farther into the forest, let's try searching in this area a bit more."""
```
Comment (Frey8914) [inline Translated/Story/08-09 Finding Bruder/msq_011418.csv:1]: further would probably sound better here

**Pull request merged**